lib/data/parse-support.js

```diff
@@ -0,0 +1,28 @@
+/// <reference path="../../types/aria-at-csv.js" />
```

Comment:
I'm seeing several empty files, lib/data/csv.js, lib/data/file.js and lib/data/parsed.js - do you know what that's about?

Reply:
Ah. I'll remove those. They were a step in a process I want to continue moving functions out of `scripts/create-example-tests.js`